recipes-browser/chromium/chromium-browser-gn.inc

```diff
@@ -1,6 +1,7 @@
 include chromium.inc
 
 DESCRIPTION = "Chromium browser"
+DEPENDS += "gn-native ninja-native"
```

Comment:
ninja-native is already set in chromium.inc:
https://github.com/Igalia/meta-browser/blob/ozone/wayland/20172906/recipes-browser/chromium/chromium.inc#L2